packages/components/src/custom-select-control/index.js

```diff
@@ -51,6 +51,7 @@ export default function CustomSelectControl( {
 	label,
 	options: items,
 	onChange: onSelectedItemChange,
+	isSmall = true,
```

Comment:
To be fair,  I'm still not sure we should even expose this on the CustomSelectControl as it's just a select and SelectControl, for instance, doesn't have that option.